modules/mod_clamav/support/z_clamav_msoffice.erl

```diff
@@ -0,0 +1,48 @@
+%% @author Marc Worrell <marc@worrell.nl>
+%% @copyright 2019 Marc Worrell
+%% @doc Additional filtering of MS Office files.
+
+%% Copyright 2019 Marc Worrell
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(z_clamav_msoffice).
+
+-export([
+    scan_file/3
+]).
+
+scan_file(File, <<"application/vnd.openxmlformats-officedocument.", _/binary>>, Context) ->
+    case {z_config:get(clamav_reject_msoffice_external_links, true),
```

Comment:
Should this not be converted to bool?

Reply:
It is a `zotonic.config` setting, question if we should be strict in typing there or not. From the site-config I usually am converting, as that is more-or-less user-input.